apps/chat-e2e/src/core/dialSharedWithMeFixtures.ts

```diff
@@ -73,6 +81,8 @@ const dialSharedWithMeTest = dialTest.extend<{
   additionalShareUserFolderPrompts: FolderPrompts;
   additionalShareUserFolderDropdownMenu: DropdownMenu;
   additionalShareUserSharedWithMeFolderDropdownMenu: DropdownMenu;
+  additionalShareUserAttachmentDropdownMenu: DropdownMenu;
+  additionalShareUserConversationData: ConversationData;
```

Comment:
I believe you can reuse the `conversationData` class without instantiating a new one, as it is not associated with the browser context

Reply:
done